src/utils/network.ts

```diff
@@ -26,12 +27,14 @@ export const getNetwork = (networkId?: number | string): Network => {
 };
 
 export const getNetworkRpc = (network: Network): string => {
-  const { WEB3_PROVIDER_HTTPS, ARBITRUM_PROVIDER_HTTPS } = getConfig();
+  const { WEB3_PROVIDER_HTTPS, ARBITRUM_PROVIDER_HTTPS, GOERLI_PROVIDER_HTTPS } = getConfig();
```

Comment:
is WEB3_PROVIDER always only for mainnet or is it a strictly year component? Why wouldn't it be able to support goerli?

Reply:
Yes, `WEB3_PROVIDER_HTTPS` is always only for mainnet.

Reply:
fixed

Reply:
fixed